doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -0,0 +1,495 @@
+        DPU_2_T           38.5      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_3_T         38.813      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+     FAN_Sensor         23.201      100.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+ MB_PORT_Sensor         21.813       97.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+MB_TMP421_Local          26.25      135.0      -5.0           140.0          -10.0      False  20230728 06:39:18
+       SSD_Temp           40.0       80.0      -5.0            83.0          -10.0      False  20230728 06:39:18
+   X86_CORE_0_T           37.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+   X86_CORE_1_T           37.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+   X86_PKG_TEMP           41.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+```
+
+show platform fan - shows the fan speed and status on the switch
+```
+root@sonic:~#show platform fan
+
+  Drawer    LED           FAN    Speed    Direction    Presence    Status          Timestamp
+--------  -----  ------------  -------  -----------  ----------  --------  -----------------
+     N/A    N/A     PSU0.fan0      50%          N/A     Present        OK  20230728 06:41:18
+     N/A    N/A     PSU1.fan0      50%          N/A     Present        OK  20230728 06:41:18
+fantray0    N/A  fantray0.fan      55%       intake     Present        OK  20230728 06:41:17
+fantray1    N/A  fantray1.fan      56%       intake     Present        OK  20230728 06:41:17
+fantray2    N/A  fantray2.fan      56%       intake     Present        OK  20230728 06:41:17
+fantray3    N/A  fantray3.fan      56%       intake     Present        OK  20230728 06:41:17
+```
+show reboot-cause history on the switch shows the dpu reboot-cause as well.
+```
+root@sonic:~#show reboot-cause history
+
+Device          Name                    Cause                       Time                                User    Comment
+
+switch          2023_10_20_18_52_28     Watchdog:1 expired;         Wed 20 Oct 2023 06:52:28 PM UTC     N/A     N/A
+switch          2023_10_05_18_23_46     reboot                      Wed 05 Oct 2023 06:23:46 PM UTC     user    N/A
+DPU0            2023_10_04_18_23_46     Power Loss                  Tue 04 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_03_18_23_46     Watchdog: stage 1 expired;  Mon 03 Oct 2023 06:23:46 PM UTC     N/A     N/A
+DPU3            2023_10_02_18_23_46     Host Power-cycle            Sun 02 Oct 2023 06:23:46 PM UTC     N/A     Host lost DPU
+DPU3            2023_10_02_17_23_46     Host Reset DPU              Sun 02 Oct 2023 05:23:46 PM UTC     N/A     N/A
+DPU2            2023_10_02_17_20_46     reboot                      Sun 02 Oct 2023 05:20:46 PM UTC     admin   User issued 'reboot'
```

Comment:
@rameshraghupathy can you elaborate how NPU is able to fetch DPU reboot cause. Can we have the DB schema for this ?

Reply:
Done